### PR TITLE
Allow inherit ligations from `calt`

### PR DESCRIPTION
**Pull request by alexeyten**
Add `calt` to `hives` so `inherit = 'calt'` works.

Also get rid of `optInBuildup`/`optOutBuildup` because they just merged into `para.ligation.defaultBuildup`.

### CODE REVIEW

**Comment by be5invis** [comment]
Well in ligation settings you are *creating* a "calt" configuration.
It would be better to inherit "calt-default" instead.

**Comment by be5invis** [comment]
Also I think it may worth to rename these internal "hives"' prefixes, like into `ligset-inherit-<>`, `ligset-enable-<>` and `ligset-disable-<>`.

File: font-src/support/ligation-data.js [hunk 1]
```diff
@@ -17,18 +16,12 @@ module.exports = function applyLigationData(data, para, argv) {
...
-		if (!comp.isOptOut) {
-			hives["ligset-" + gr] = { caltBuildup: ligSets };
-		}
+		defaultBuildup[comp.tag] = ligSets;
+		hives["ligset-" + gr] = { caltBuildup: ligSets };
 	}
 
 	para.ligation = {
-		defaultBuildup: { ...optInBuildup, ...optOutBuildup },
+		defaultBuildup: defaultBuildup,
```
Comment (be5invis) [inline font-src/support/ligation-data.js:24]: No need to duplicate this variable name. Use `{ defaultBuildup }`.

File: font-src/support/ligation-data.js [hunk 2]
```diff
@@ -17,18 +16,12 @@ module.exports = function applyLigationData(data, para, argv) {
...
-		if (comp.isOptOut) {
-			optOutBuildup[comp.tag] = ligSets;
-		} else {
-			optInBuildup[comp.tag] = ligSets;
-		}
-		if (!comp.isOptOut) {
-			hives["ligset-" + gr] = { caltBuildup: ligSets };
-		}
+		defaultBuildup[comp.tag] = ligSets;
+		hives["ligset-" + gr] = { caltBuildup: ligSets };
```
Comment (be5invis) [inline font-src/support/ligation-data.js:20]: We may need to rename these internal hive names, like into `ligset-inherit-<>`, `ligset-enable-<>` and `ligset-disable-<>`.

**Comment by alexeyten** [comment]
> Well in ligation settings you are _creating_ a "calt" configuration.
> It would be better to inherit something like "calt-original" or "calt-built-in" instead.

I'm not sure how to do it right.

Maybe we should inherit `calt` by default and have something like `inherit: 'none'` to clean it up.

**Comment by be5invis** [comment]
@alexeyten Inheritance should be opt-in.
An interesting solution is that to rename `[composite.calt]` into `[composite.calt-builtin]` in `params/ligation-set.toml`. OT tags are stored as properties so no worries about tag compatibility.

**Comment by alexeyten** [comment]
Maybe `[composite.default]` since it is the default.

**Comment by be5invis** [comment]
@alexeyten `default-calt` seems better.

**Pull request merged**